contribs/gmf/src/controllers/defaultConfig.js

```diff
@@ -0,0 +1,15 @@
+goog.provide('gmf.defaultConfig');
```

Comment:
Shouldn't we add:
```
// webpack: import 'jquery'; 
// webpack: import 'angular'; 
// webpack: import 'angular-gettext';
// webpack: import 'angular-dynamic-locale';
```

Reply:
What do you think @ger-benjamin

Reply:
Yes I think... I'm not really sure about webpack with our application for now. But That will not break anything. So let's do it.